.gitignore

```diff
@@ -1,6 +1,3 @@
-## Ignore Visual Studio temporary files, build results, and
-## files generated by popular Visual Studio add-ons.
```

Comment:
Oh, I thought it was supposed to be a section header (like "User-specific files")